native/python/setup.py

```diff
@@ -167,7 +166,7 @@ def get_inputs(self) -> List[str]:
         "tbb>=2021.8;platform_machine=='x86_64'",
         # PyTorch has no ABI compatibility between releases; this means we have
         # to ensure that we depend on the exact same version that we used to
-        # build our extension module.
-        "torch==" + version.parse(torch.__version__).public,
+        # build fairseq2n.
+        "torch==" + torch.__version__,
```

Comment:
why did we abandon the version.parse() way ? At least for me the old code works and the new one doesn't. I installed torch  via pip and get the version listed as '2.0.1' and not '2.0.1+cu117',

Reply:
Hey @antoine-tran, thanks for the report! Working on a fix now. With pip-based Python virtual environments it is the opposite case. So looks like I have to have handle conda and pip packages of PyTorch separately to have a full working solution.